apps/etterlatte-saksbehandling-ui/client/src/components/oppgavebenk/tildeling/VelgSaksbehandler.tsx

```diff
@@ -0,0 +1,162 @@
+        byttSaksbehandler(
+          { oppgaveId, type, nysaksbehandler: { saksbehandler: selectedSaksbehandler.ident!, versjon } },
+          (result) => {
+            oppdaterTildeling(oppgaveId, selectedSaksbehandler.ident, result.versjon)
+            setValgtSaksbehandler(saksbehandler)
+            setOpenDropdown(false)
+          },
+          (error) => console.log(error)
+        )
+      }
+    }
+  }
+
+  const onTildelTilMeg = () => {
+    tildelSaksbehandler(
+      { oppgaveId, type, nysaksbehandler: { saksbehandler: innloggetSaksbehandler.ident, versjon } },
+      (result) => {
+        oppdaterTildeling(oppgaveId, innloggetSaksbehandler.ident, result.versjon)
+        setValgtSaksbehandler({ navn: innloggetSaksbehandler.navn, ident: innloggetSaksbehandler.ident })
+        setOpenDropdown(false)
+      },
+      (error) => {
+        console.log(error)
+      }
+    )
+  }
+
+  const onFjernTildeling = () => {
+    fjernSaksbehandler(
+      { oppgaveId, sakId, type, versjon },
+      (result) => {
+        oppdaterTildeling(oppgaveId, null, result.versjon)
+        setValgtSaksbehandler(undefined)
+        setOpenDropdown(false)
+      },
+      (error) => console.log(error)
+    )
+  }
+
+  return (
+    <div>
+      {erRedigerbar ? (
+        <Dropdown open={openDropdown}>
+          <Button
+            as={Dropdown.Toggle}
+            icon={valgtSaksbehandler?.ident ? <PersonPencilIcon /> : <PersonPlusIcon />}
+            iconPosition="left"
+            size="small"
+            variant="tertiary"
+            onClick={() => setOpenDropdown(true)}
+          >
+            {valgtSaksbehandler?.navn
+              ? `${valgtSaksbehandler.navn} ${valgtSaksbehandler.navn === innloggetSaksbehandler.navn ? '(meg)' : ''}`
+              : 'Ikke tildelt'}
+          </Button>
+          <DropdownMeny onClose={() => setOpenDropdown(false)}>
+            <div>
+              <VelgSaksbehandlerCombobox
+                label="Velg saksbehandler"
+                options={saksbehandlereIEnhet.map((behandler) => behandler.navn!)}
+                onToggleSelected={onSaksbehandlerSelect}
+                selectedOptions={!!valgtSaksbehandler ? [valgtSaksbehandler.navn!] : []}
+              />
+              {!valgtSaksbehandler?.ident?.includes(innloggetSaksbehandler.ident) && (
+                <ValgButton variant="tertiary" size="xsmall" onClick={onTildelTilMeg}>
+                  Tildel til meg
+                </ValgButton>
+              )}
+            </div>
+            {valgtSaksbehandler?.ident && (
+              <div>
+                <ValgButton
+                  variant="secondary"
+                  size="small"
+                  onClick={onFjernTildeling}
+                  icon={<PersonCrossIcon />}
+                  iconPosition="right"
+                >
+                  Fjern tildeling
+                </ValgButton>
+              </div>
+            )}
+          </DropdownMeny>
+        </Dropdown>
```

Comment:
Kunne med fordel lagt til sjekk på `isPending` på knappene, eventuelt hele dropdown-greia. Kallet går sykt kjapt, men greit å gjøre det vi kan for å forhindre saksbehandler fra dobbeltklikk eller lignende som kan skape støy i loggene.